modules/gapi/include/opencv2/gapi/backends/gbackend.hpp

```diff
@@ -19,7 +19,7 @@
 #include "opencv2/gapi/util/optional.hpp"
 #include "opencv2/gapi/own/scalar.hpp"
 
-#include "compiler/gmodel.hpp"
+#include "opencv2/gapi/compiler/gmodel.hpp"
```

Comment:
should this file really be part of OpenCV public headers (included in the installation)?

I believe what we aligned on is that G-API backends are built with OpenCV as modules (with full access to G-API's internal headers)

Reply:
Yes, you are correct. Reworked, now there are no changes on header export in this PR (only symbols and some cleanup and general header restructuring)